### PR TITLE
BookStore

### PR DESCRIPTION
**Pull request by PierreBarba**
# Hello reviewer 😄 , I hope you have a great day 🌴 , thanks for the support any suggestions are welcome
* for this review I created some files and gave a wireframe first wireframe
## JavaScript requirements

> Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

## general requirements

> Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
> Make sure that you used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
> Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## Project requirements

> Initialise React app.
> The building blocks of your app should be set as reusable components. Create a directory for your components.
> Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
> Books - the default view:
> Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
> Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
> Should display a form for adding a book (no functionality yet).
> Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
> Categories:
> Should display a button "Check status" only.
> Styling is not required at this point.

## workflow
- create a new folder
- create new files
- import the files
- create the routs 
- add book section
- add form section
- fix the linter error

### CODE REVIEW

**Review by christianbiring1 — COMMENTED** [review]
Hi @PierreBarba , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid


You have submitted a project, that has no linters set up, kindly make sure your linters checks are running before you resubmit for another review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,187 @@
...
+- *you can watch diferents books.*
+- *Use React.*
+- *Use Redux.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="in progres..." target="_blank" rel="noopener noreferrer">math-magician</a>
```
Comment (christianbiring1) [inline README.md]: [OPTIONAL]
- [x] You may consider changing the text of the anchor tag as well. `Book store` instead of `Math magician`

**Review by christianbiring1 — APPROVED** [review]
Hi @PierreBarba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
congraaaa

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Professional README file :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (PierreBarba) [reply README.md]: thanks i will do that

**Pull request merged**